cafebabel/tests/test_base.py

```diff
@@ -21,12 +21,6 @@ def test_homepage_is_displaying(client):
     assert '<meta name=twitter:card content="summary">' in response
 
 
```

Comment:
erratum: commit name should have been `remove associated test test_homepage_contains_published_articles`